Tribler/Policies/BoostingManager.py

```diff
@@ -0,0 +1,676 @@
+# -*- coding: utf-8 -*-
+# Written by Egbert Bouman, Mihai Capotă, Elric Milon, and Ardhi Putra Pratama H
+"""Manage boosting of swarms"""
+import ConfigParser
+import json
+import logging
+import os
+import random
+from binascii import hexlify, unhexlify
+
+import errno
+import libtorrent as lt
+
+from Tribler.Core.DownloadConfig import DownloadStartupConfig
+from Tribler.Core.TorrentDef import TorrentDef
+from Tribler.Core.Utilities.install_dir import determine_install_dir
+from Tribler.Core.simpledefs import DLSTATUS_SEEDING, NTFY_TORRENTS, NTFY_UPDATE, NTFY_CHANNELCAST
+from Tribler.Main.globals import DefaultDownloadStartupConfig
+from Tribler.Policies.BoostingSource import ChannelSource
+from Tribler.Policies.BoostingSource import DirectorySource
+from Tribler.Policies.BoostingSource import RSSFeedSource
+from Tribler.dispersy.taskmanager import TaskManager
+
+NUMBER_TYPES = (int, long, float)
+
+# CONFIG_FILE = "boosting.ini"
+TRIBLER_ROOT = determine_install_dir()
+CONFIG_FILE = os.path.join(TRIBLER_ROOT, "boosting.ini")
+
+
+def lev(trnt_1, trnt_2):
+    """
+    Calculates the Levenshtein distance between a and b.
```

Comment:
'Calculates the Levenshtein distance between two torrents'.

Can you add an example? If I see this code for the first time, I have no idea about the structure of `trnt_1` and `trnt_2`.